20230728/s15-1690548412616/README.md

```diff
@@ -0,0 +1,69 @@
+# Scenario 15 - ALS Baseline with Sims-only, Disabled JSON.stringify [ALS v14.2.3](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.2.3) + Scale 4 + 6x k6 VUs + Enhanced Caching for ValidateParticipants | 1 | N | [account-lookup-service/pull/461](https://github.com/mojaloop/account-lookup-service/pull/461), [v14.2.4-snapshot.3](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.2.4-snapshot.3)
```

Comment:
Can this description be consistent with the rest of the scenario descriptions? What does  | 1 | N | represent?